armi/reactor/components/basicShapes.py

```diff
@@ -20,8 +20,7 @@
 """
 import math
 
-from armi.reactor.components import ShapedComponent
-from armi.reactor.components import componentParameters
+from armi.reactor.components.component import ShapedComponent, componentParameters
```

Comment:
Mike, can we fix your IDE?

I don't particularly like these changes, and they make your PRs more muddled to read. Like... you didn't actually touch this file.

#sorry

Reply:
Sure thing. I was able to turn it off, so I'll go through and revert these import changes.